css/style.css

```diff
@@ -1,9 +1,101 @@
+/* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+body {
+  font-family: "Roboto", sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
 
- /* We are using the 'Roboto' font from Google. This has already been added to your HTML header */
+#root {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  width: 100%;
+  height: 100%;
+}
+#root > * {
+  width: 60%;
+}
+header #logo {
+  max-height: 2em;
+}
 
-body {
-    font-family: 'Roboto', sans-serif;
-    -webkit-font-smoothing: antialiased;
+#header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  margin: 0.5em 0;
+  font-size: 1.5em;
+  color: grey;
+}
+
+#header-container nav a:hover {
```

Comment:
`#header-container` is only used for making some "flex wrapper" around our <img> (logo image) and <nav>. 
<nav> is unique at our page, so when styling whatever inside it, you can omit #header-container in its css selector, and write just this:
```css
nav a:hover {
 ...
}
```